packages/legacy-workbench/src/js/angular/core/services/repositories.service.js

```diff
@@ -277,14 +277,8 @@ repositories.service('$repositories', ['toastr', '$rootScope', '$timeout', '$loc
             return this.getRepositories().find((repository) => repository.id === repositoryId);
         };
 
-        this.getReadableRepositories = function () {
-            return _.filter(this.getRepositories(), function (repo) {
-                return $jwtAuth.canReadRepo(repo);
-            });
-        };
-
         this.getReadableGraphdbRepositories = function () {
-            return this.getReadableRepositories()
+            return this.getAllAccessibleRepositories()
```

Comment:
Although it is not done in this MR, the naming of the methods `getAllAccessibleRepositories` and `getAccessibleRepositories` is somewhat confusig.

By the name I would thing they do the same thing.